Packs/Okta/ReleaseNotes/3_1_6.md

```diff
@@ -0,0 +1,4 @@
+
+#### Playbooks
+##### Okta - User Investigation
+- Update parameter on condition task 13 "Found results from Okta?"
```

Comment:
Fixed an issue with a condition input of task "Found results from Okta?"

Reply:
Done